src/server/local/index.js

```diff
@@ -0,0 +1,152 @@
+import path from 'path';
+import expressStaticGzip from 'express-static-gzip';
+
+import {
+  articleDataPath,
+  cpsAssetPageDataPath,
+  frontPageDataPath,
+  IdxDataPath,
+  legacyAssetPageDataPath,
+  mostReadDataRegexPath,
+  onDemandRadioDataPath,
+  onDemandTvDataPath,
+  secondaryColumnDataRegexPath,
+} from '#app/routes/utils/regex';
+import { LOCAL_SENDFILE_ERROR } from '#lib/logger.const';
+import nodeLogger from '#lib/logger.node';
+import constructDataFilePath from './constructDataFilePath';
+
+const logger = nodeLogger(__filename);
+
+/*
+ * Local env routes - fixture data
+ */
+const sendDataFile = (res, dataFilePath, next) => {
+  res.sendFile(dataFilePath, {}, sendErr => {
+    if (sendErr) {
+      logger.error(LOCAL_SENDFILE_ERROR, { error: sendErr });
+      next(sendErr);
+    }
+  });
+};
+
+const PUBLIC_DIRECTORY = 'build/public';
+
+export default server => {
+  server
+    .use((req, res, next) => {
+      if (req.url.substr(-1) === '/' && req.url.length > 1)
+        res.redirect(301, req.url.slice(0, -1));
+      else next();
+    })
+    .use(
+      expressStaticGzip(PUBLIC_DIRECTORY, {
+        enableBrotli: true,
+        orderPreference: ['br'],
+        redirect: false,
+      }),
+    )
+    .get(articleDataPath, async ({ params }, res, next) => {
+      const { service, id, variant } = params;
+
+      const dataFilePath = constructDataFilePath({
+        pageType: 'articles',
+        service,
+        id,
+        variant,
+      });
+
+      sendDataFile(res, dataFilePath, next);
+    })
+    .get(frontPageDataPath, async ({ params }, res, next) => {
+      const { service, variant } = params;
+
+      const dataFilePath = constructDataFilePath({
+        pageType: 'frontpage',
+        service,
+        variant,
+      });
+
+      sendDataFile(res, dataFilePath, next);
+    })
+    .get(mostReadDataRegexPath, async ({ params }, res, next) => {
+      const { service, variant } = params;
+      const dataFilePath = constructDataFilePath({
+        pageType: 'mostRead',
+        service,
+        variant,
+      });
+
+      sendDataFile(res, dataFilePath, next);
+    })
+    .get(onDemandRadioDataPath, async ({ params }, res, next) => {
+      const { service, serviceId, mediaId } = params;
+
+      const dataFilePath = path.join(
```

Comment:
Just a note for future refactoring (which you will have spotted too) - OD Radio & OD TV construct the file path in this file, rather than using `constructDataFilePath`.

Reply:
I've made a note - something we could pick up within the team 👍